AzureDevOps.Authentication/Src/MsaAuthentication.cs

```diff
@@ -33,8 +33,7 @@ namespace AzureDevOps.Authentication
 
     public sealed class MsaAuthentication : Authentication, IMsaAuthentication
     {
-        public const string DefaultAuthorityHost = AzureDevOps.Authentication.Authority.AuthorityHostUrlBase + "/live.com";
-        internal const string QueryParameters = "domain_hint=live.com&display=popup&site_id=501454&nux=1";
+        public const string DefaultAuthorityHost = AzureDevOps.Authentication.Authority.AuthorityHostUrlBase + "/common";
```

Comment:
nit: I see you already have `AzureDevOps.Authentication.Authority.DefaultAuthorityHostUrl` defined with the required `common` value. you can consider using it directly.